manifests/params.pp

```diff
@@ -302,7 +382,7 @@
           $service_config_perms = '0600'
         }
         default: {
-          if $majdistrelease <= '5' {
+          if versioncmp($majdistrelease, '5') <= 0 {
```

Comment:
versioncmp is part of Puppet since at least 3.5:

https://docs.puppet.com/puppet/3.5/function.html#versioncmp